Core/Database/Player.cs

```diff
@@ -33,15 +31,15 @@ public class Player : BaseModel, ICache
         public uint GivenStatusPoints { get; set; }
         public uint AvailableStatusPoints { get; set; }
 
-        public static async Task<Player> GetPlayer(IDbConnection db, ICacheManager cacheManager, Guid account, byte slot)
+        public static async Task<Player> GetPlayer(IPlayerRepository repository, ICacheManager cacheManager, Guid account, byte slot)
```

Comment:
With this new repository setup maybe it's worth considering giving the player an instance of both the `IPlayerRespoitory` and and the `CacheManager` and any other external systems needed. This would result in the methods here only getting data they need to function and no dependencies which in my opinion makes for cleaner and more maintainable code.
While implementing you had to exchange `IDbConnection` for `IPlayerRepository` in multiple placer the mentioned approach would greatly decrease future refactoring effort.

However, I'm not sure about potential performance implications of this.